SimTKcommon/src/ParallelExecutor.cpp

```diff
@@ -29,6 +29,11 @@
 #include <algorithm>
 #include <mutex>
 
+#if defined(__FreeBSD__)
```

Comment:
Consider using a consistent style. `#ifdef ...`

Reply:
```#if defined()``` is more generic, another clauses can be easily added to it, like ```#if defined(A) || defined(B)```.